packages/contracts-bedrock/snapshots/abi/OptimismSuperchainERC20.json

```diff
@@ -629,6 +629,11 @@
     "name": "TotalSupplyOverflow",
     "type": "error"
   },
+  {
+    "inputs": [],
+    "name": "ZeroAddress",
+    "type": "error"
+  },
```

Comment:
These lines are repeated below, were them autogenerated? Are we sure we need them or could be removed?

Reply:
this is autogenerated, but if this is repeated then we may have the error in both the SuperchainERC20 and the OptimismSuperchainERC20, will check and remove accordingly